pkg/ginkgo_helper/metrics.go

```diff
@@ -0,0 +1,113 @@
+package ginkgo_helper
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	"github.com/pingcap/test-infra/sdk/resource"
+	"github.com/prometheus/client_golang/api"
+	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/common/model"
+)
+
+func BuildPrometheusAPI(tc resource.TiDBCluster) (v1.API, error) {
+	url, err := tc.ServiceURL(resource.Prometheus)
```

Comment:
I think we should not import infra sdk in other pacakges.

Reply:
We could let user pass in the result of `ServiceURL` directly.

Reply:
Yeah!